packages/react-data-grid/src/HeaderGroupCell.js

```diff
@@ -0,0 +1,80 @@
+const React          = require('react');
+const ReactDOM      = require('react-dom');
+const joinClasses    = require('classnames');
+const ExcelColumn    = require('./PropTypeShapes/ExcelColumn');
+require('../../../themes/react-data-grid-header.css');
+
+import PropTypes from 'prop-types';
+
+function simpleCellRenderer(group: {name: string}): ReactElement {
+  let headerText = group.get('name');
+  return <div className="react-grid-HeaderGroupCell__value">{headerText}</div>;
+}
+
+class HeaderGroupCell extends React.Component {
```

Comment:
Do we need this component? What differences does it provide to HeaderCell? Can that component be resused and we pass in what in any customisation as props?